LocalSettings.php

```diff
@@ -4684,7 +4688,7 @@
 		'chromiumrosewiki' => "//$wmgUploadHostname/chromiumrosewiki/b/ba/ChromiumRose135x135.png",
 		'clementsworldbuildingwiki' => "//$wmgUploadHostname/clementsworldbuildingwiki/3/39/Cw_logo.png",
 		'cnvwiki' => "//$wmgUploadHostname/cnvwiki/b/bc/Wiki.png",
-		'coldbloodedwiki' => "//$wmgUploadHostname/coldbloodedwiki/3/35/WikiLogo.png",
+		'coldbloodedwiki' => "//i0.wp.com/static.miraheze.org/coldbloodedwiki/3/35/WikiLogo.png?w=135&ssl=1",
```

Comment:
We do not allow external sites for logos and this one doesn't resolve anyway, please make it $wmgUploadHostname